Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -333,15 +334,47 @@ WarpXOpenPMDPlot::WriteOpenPMDParticles (const amrex::Vector<ParticleDiag>& part
        // integer attribs, and it is automatically dumped as particle record
        // when ionization is on.
        int_flags.resize(1, 1);
+       tmp.AddIntComp(false);
     }
 
-    // Convert momentum to SI
-    pc->ConvertUnits(ConvertDirection::WarpX_to_SI);
+#ifdef WARPX_QED
+      if( pc->has_breit_wheeler() ) {
```

Comment:
@lucafedeli88 @NeilZaim I think I forgot some QED attributes here in the past for openPMD output. This PR should also add support for those now.